.github/workflows/pkgci_regression_test.yml

```diff
@@ -174,7 +181,12 @@ jobs:
       SDXL_CLIP_CONFIG_FILE_PATH: build_tools/pkgci/external_test_suite/${{ matrix.sdxl-clip-config-file }}
       SDXL_VAE_CONFIG_FILE_PATH: build_tools/pkgci/external_test_suite/${{ matrix.sdxl-vae-config-file }}
       VENV_DIR: ${{ github.workspace }}/venv
+      LD_LIBRARY_PATH: ~/Python-3.11.9
     steps:
+      - name: Pre Checkout MI300 Step
+        if: contains(matrix.name, 'gfx942')
+        run: |
+          sudo chmod -R 777 ~/actions-runner/_work
```

Comment:
Why was/is this needed? Is this something we could configure on the runner machine itself instead of having a job step for it?

Reply:
Every time a new file is created in this directory, it is being write protected. So when the CI tries to clean up the working directory and run on the next time a PR is created, it is getting denied. Not exactly sure if I can do something to make anything there or created in the future under this directory to not be write detected.

Reply:
Strange. Well, I don't care enough right now to track down a fix that is transparent to the workflow file :P

Could add a comment with that context so we can follow up later.

Reply:
Yeah, there is probably a way. Will put down a comment